docs/inflight_exit_scenarios.md

```diff
@@ -0,0 +1,415 @@
+        "last_mined_child_block_timestamp": 1535031020,
+        "last_mined_child_block_number": 11000,
+        "eth_syncing": true,
+        "byzantine_events": [
+            {
+                "event": "block_withholding",
+                "details": {
+                    "blockhash"  : "DB32876CC6F...",
+                    "blocknum"  : 10000,
+                }
+            }
+        ]
+    }
+}
+ ```
+She notices that the chain is byzantine and the transaction she just submitted was not included in a block (therefore it's an in-flight transaction)
+TODO - How does Alice know that her transaction hasn't been included? Does she have to store the details of every transaction she submits until it is put into a block?
+
+Alice starts an in-flight exit.
+
+#### 1. Get the exit data
+`/inflight_exit.get_data` 
+```json
+ {
+    "txbytes": "F3170101C0940000..."
+ }
+ ```
+
+ response:
+```json
+ {
+    "data": {
+        "txbytes": "F3170101C0940000...",
+        "sigs": "7C29FB8327F60BBFC62...",
+        "input_txs" : [
+            "F81891018080808...",
+            "2A0341808602A01..."
+        ],
+        "input_proofs" : [
+             "CEDB8B31D1E4C...",
+             "A67131D1E904C..."
+        ]
+    }
+ }
+ ```
+
+#### 2. Start the IFE
+```
+RootChain.startInFlightExit(
+    response.data.txbytes, 
+    response.data.input_txs, 
+    response.data.input_proofs, 
+    response.data.sigs, 
+    {"value": inFlightExitBond}
+)
+```
+
+#### 3. Check status again 
+```json
+{
+    "version": "1",
+    "success": true,
+    "data": {
+        "last_validated_child_block_number": 10000,
+        "last_mined_child_block_timestamp": 1535031020,
+        "last_mined_child_block_number": 11000,
+        "eth_syncing": true,
+        "byzantine_events": [
+            {
+                "event": "block_withholding",
+                "details": {
+                    "blockhash"  : "DB32876CC6F...",
+                    "blocknum"  : 10000,
+                }
+            },
+            {
+                "event": "piggyback_available",
+                "details": {
+                    "txbytes": "F3170101C0940000...",
+                    "available_outputs" : [
+                        {"index": 0, "address": "0x7890..."},
+                        {"index": 1, "address": "0x1234..."},
+                    ]
+                }
+            },
+        ],
+        "inflight_exits": [
+            {
+                "txhash": "230C450180808080...",
+                "txbytes": "F3170101C0940000...",
+                "eth_height" : 615441,
+            }
+        ]
+    }
+}
+ ```
+
+ Alice sees that her in-flight exit is in progress and she can now piggyback her ouput. Note that as Alice is the sole owner of the inputs, she does not need to piggyback any input.
+
+#### 4. Piggyback the output
+The second argument is `5` because she is piggybacking the first output.
+```
+RootChain.piggybackInFlightExit(
+    response.data.txbytes,
+    5, 
```

Comment:
I fear that the `+4`  part will bite us. Maybe the solution to consider is to take an analogous path to `challengeInFlightExitOutputSpent` vs its `input` version.

We'd then just have `RootChain.piggybackInFlightExitOutput`/`-Input`, which would do the `+4`, and the `4` of that is kept in contract's settings?  CC @paulperegud 

Then the data from `status/` would be usable "as-is" in the contract